subsys/bluetooth/host/conn.c

```diff
@@ -136,6 +137,8 @@ static void notify_connected(struct bt_conn *conn)
 			cb->connected(conn, conn->err);
 		}
 	}
+
```

Comment:
We should probably do something like:
if (!conn->err) {
    bt_gatt_connected(conn);
}